lib/scanners/base.js

```diff
@@ -34,17 +34,41 @@ var BaseScanner = function (beaconMapper, beaconRegionMapper, beaconRegionMonito
     this.init = function (networkIdentifier) {
         if (!_.isUndefined(networkIdentifier)) {
             self.networkId = networkIdentifier;
+            var hasPermissions = function () {
```

Comment:
`hasPermissions` to me sounds like a function that returns whether or not something has permissions. You're not returning anything from this function, so it clearly does not do that. What I'm trying to say is that the naming seems to be off.